webui/react/src/e2e/models/hew/Dropdown.ts

```diff
@@ -1,22 +1,2 @@
-import { BaseComponent } from 'e2e/models/BaseComponent';
-
-/**
- * Returns a representation of the Mondal component from Hew.
- * This constructor represents the contents in hew/src/kit/Dropdown.tsx.
- * @param {object} obj
- * @param {CanBeParent} obj.parent - The parent used to locate this Dropdown
- * @param {string} obj.selector - Used instead of `defaultSelector`
- */
-export class Dropdown extends BaseComponent {
-  /**
-   * Returns a templated selector for children components.
-   * @param {string} id - menu item id
-   */
-  static selectorTemplate(id: string): string {
-    return `li.ant-dropdown-menu-item[data-menu-id$="${id}"]`;
-  }
-  protected readonly _menu = new BaseComponent({
-    parent: this.root,
-    selector: '.ant-dropdown ul.ant-dropdown-menu:visible',
-  });
-}
+export { Dropdown as DropdownMenu } from 'e2e/models/ant/Dropdown';
```

Comment:
should we just delete these files? I don't really see what value they add.

Reply:
it's so we can show a 1-to-1 mapping of files with determined ui. hew dropdown does this, so we do too.
```
  return content ? (
    <AntdPopover
      className={className}
      content={content}
      open={open}
      overlayClassName={themeClass}
      overlayStyle={overlayStyle}
      placement={placement}
      showArrow={false}
      trigger="click"
      onOpenChange={onOpenChange}>
      {children}
    </AntdPopover>
  ) : (
    <AntDropdown
      className={className}
      disabled={disabled}
      menu={antdMenu}
      open={open}
      overlayClassName={themeClass}
      overlayStyle={overlayStyle}
      placement={placement}
      trigger={[isContextMenu ? 'contextMenu' : 'click']}
      onOpenChange={onOpenChange}>
      {children}
    </AntDropdown>
  );
```